mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -51,48 +81,71 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 	METHOD method_8219 getProtectionAmount (Ljava/lang/Iterable;Lnet/minecraft/class_1282;)I
 		ARG 0 equipment
 		ARG 1 source
-	METHOD method_8220 accept (Lnet/minecraft/class_1890$class_1891;Lnet/minecraft/class_1799;)V
-		ARG 0 enchantmentHandler
+	METHOD method_8220 process (Lnet/minecraft/class_1890$class_1891;Lnet/minecraft/class_1799;)V
```

Comment:
```suggestion
	METHOD method_8220 forEachEnchantment (Lnet/minecraft/class_1890$class_1891;Lnet/minecraft/class_1799;)V
```
This method is quite similar to the one above, applying an action to every enchantment on the stack.

Reply:
It is like for only one item stack i think 🤔

Reply:
Done.